test/e2e-rh-sdl/rapidast/far/create_rapidast_configmap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
nitpick: Using `#!/usr/bin/env bash` as the shebang is encouraged for portability

see https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability

Reply:
Done